CHANGELOG.md

```diff
@@ -1,12 +1,29 @@
 # CHANGELOG
 
-## v0.12.0-rc.0
+## v0.12.0
+*February 24th, 2022*
 
-> ⚠️  This is a release candidate and not the final release for v0.12.0.
+This release notably brings compatibility with Cosmos SDK 0.45 and IBC v3.0.0-rc.0,
+as well as support for non-standard ports in the channel handshake.
+It also contains a fix for a bug where `SendPacket` events were duplicated when emitted at EndBlock,
+and fixes another bug where Hermes would clear packet at startup even when `clear_on_start = false`.
+The relayer will now also honor the `tracing` filter specified in the `RUST_LOG` environment variable, if any.
+
+### Note for operators
+
+As of this release, the relayer will not respond to the `SIGHUP` signal and will therefore
+not reload the configuration anymore. This feature has been deemed unnecessary given the
+recent performance improvements, and it is now recommended to just restart the relayer
+when the configuration is updated.
+
+Additionally, a new CLI command [`clear packets`](https://hermes.informal.systems/commands/relaying/clear.html)
```

Comment:
Note that the final changelog now diverges from .changelog/v0.12.0/summary.md (because of the `clear packets` remark).